script/run-unit-test-help.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+const commandLineUsage = require('command-line-usage');
+
+const helpSections = [
+  {
+    header: 'Unit test run script',
+  },
+  {
+    header: 'Examples',
+    content: [
+      {
+        desc: '1. Run all the unit tests ',
+        example: '$ yarn test:unit',
+      },
+      {
+        desc: '2. Run tests for a specific folder ',
+        example: '$ yarn test:unit src/applications/vaos',
```

Comment:
I think the example for running tests for a specific folder may need to be updated. Running `yarn test:unit src/applications/vaos` alone doesn't seem to work.